### PR TITLE
Feature/button and input

### PR DESCRIPTION
**Pull request by Chrinsieboy**
Components van @Huib8 refactoren :)
All is from #12

### CODE REVIEW

File: src/resources/vue/components/terminal-scan.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	export default {
+		name: 'index',
+		components: {
+			Box,
+			Button,
+			Input
+		},
+		methods: {
+			buttonAction() {
```
Comment (DanericNetwork) [inline src/resources/vue/components/terminal-scan.vue:28]: This button action is not used?

File: src/routes/api.php [hunk 2]
```diff
@@ -17,3 +17,7 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
```
Comment (DanericNetwork) [inline src/routes/api.php:21]: also duplicate, huib also had this problem with the same route

File: src/resources/vue/components/terminal-scan.vue (hunk 1 above)
Comment (Chrinsieboy) [reply src/resources/vue/components/terminal-scan.vue]: fixed :), now it is used

**Review by Timtendo12 — APPROVED** [review]
LGTM

**Review by DanericNetwork — APPROVED** [review]

**Pull request merged**